dev/src/Command/AddComponentCommand.php

```diff
@@ -184,21 +210,215 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $composer->updateMainComposer();
         $composer->createComponentComposer($new->displayName, $new->githubRepo);
 
+        $this->validateOptions($input, $output);
+        if ($input->getOption('bazel-path')) {
+            $googleApisDir = realpath($input->getOption('bazel-path'));
+            $output->writeln("\n\nbazel build library");
+            $output->writeln($this->bazelQueryAndBuildLibrary($new->protoPath, $googleApisDir));
+            $output->writeln("\n\nCopying the library code from bazel-bin");
+            $output->writeln($this->owlbotCopyBazelBin($new->componentName, $googleApisDir));
+            $output->writeln("\n\nOwlbot post processing");
+            $output->writeln($this->owlbotPostProcessor());
+        } else if ($input->getOption('googleapis-gen-path')) {
```

Comment:
nit, but the [PSR-12 coding styles](https://www.php-fig.org/psr/psr-12/#:~:text=The%20keyword%20elseif%20SHOULD%20be,be%20on%20the%20next%20line.) tate

> The keyword elseif SHOULD be used instead of else if so that all control keywords look like single words.

```suggestion
        } elseif ($input->getOption('googleapis-gen-path')) {
```